packages/zone/src/virtual.js

```diff
@@ -13,6 +14,8 @@ import {
 
 import { Far } from '@endo/far';
 
+import { makeOnceKit } from './make-once.js';
+
 const emptyRecord = harden({});
 const initEmpty = harden(() => emptyRecord);
 
```

Comment:
For virtual zone, why is `isStorable` the same as `canBeDurable`? Even though this PR does not change that, it surprised me. I would not expect a virtual-but-not-durable exo or store to itself be `canBeDurable`, but I would expect it to be storable in a virtual zone.

Reply:
I don't fully understand the subtleties of virtual-but-not-durable.  Can you suggest a better implementation for `isStorable` that would be clearly correct?

Reply:
I found a hacky way (involving a dedicated BigStore and a try-catch) to implement the distinction in the absence of anything like `globalThis.VatData.canBeVirtual`.  And in the process, I wrapped `canBeDurable` so that `isStorable` always returns a boolean instead of ever throwing.

This is probably a controversial change, so I've isolated it in a single commit that we can refine or drop as decided in review.

Reply:
https://github.com/Agoric/agoric-sdk/pull/7971 suggests different isStorable rules, and temporarily moves the try/catch logic to a new `isPassable`

Reply:
Done